src/commands/login.ts

```diff
@@ -0,0 +1,84 @@
+import { Command } from "@oclif/core";
+import OAuthServer, { ACCOUNT_URL } from "../lib/auth/oauth-client";
+import open from "open";
+
+type AccessToken = {
+  access_token: string;
+  token_type: string;
+  ttl: string;
+  state: string;
+};
+
+export default class LoginCommand extends Command {
+  static description = "Login to your Fauna account.";
+  static examples = ["$ fauna login"];
+  static flags = {};
+
+  async run() {
+    await this.execute();
+  }
+
+  async getSession(access_token: string) {
+    const myHeaders = new Headers();
+    myHeaders.append("Authorization", `Bearer ${access_token}`);
+
+    const requestOptions = {
+      method: "POST",
+      headers: myHeaders,
+    };
+    const response = await fetch(`${ACCOUNT_URL}/session`, requestOptions);
+    if (response.status >= 400) {
+      throw new Error(`Error creating session: ${response.statusText}`);
+    }
+    const session = await response.json();
+    return session;
+  }
+
+  async listDatabases(account_key: string) {
+    const myHeaders = new Headers();
+    myHeaders.append("Authorization", `Bearer ${account_key}`);
+
+    const requestOptions = {
+      method: "GET",
+      headers: myHeaders,
+    };
+    const response = await fetch(`${ACCOUNT_URL}/databases`, requestOptions);
+    if (response.status >= 400) {
+      throw new Error(`Error listing databases: ${response.statusText}`);
+    }
+    const databases = await response.json();
+    console.log(databases);
+    return databases;
+  }
```

Comment:
is this reusable by the list databases command? if so, should we extract it into a helper?

Reply:
not reusable, just added quick for the demo. Going to break it out when working on https://faunadb.atlassian.net/browse/FE-5876